cardano-api/src/Cardano/Api/KeysByron.hs

```diff
@@ -54,6 +68,127 @@ import           Cardano.Api.SerialiseTextEnvelope
 -- This is a type level tag, used with other interfaces like 'Key'.
 --
 data ByronKey
+data ByronKeyLegacy
+
+class IsByronLegacyFormat key where
+  isLegacyFormat :: ByronKeyFormat key
+
+data ByronKeyFormat key where
+  ByronLegacyKeyFormat :: ByronKeyFormat ByronKeyLegacy
+  ByronNonLegacyKeyFormat :: ByronKeyFormat ByronKey
```

Comment:
Naming: perhaps the opposite of "legacy" is "modern"? :-)

Reply:
@dcoutts How about `PrePBFT` and `PBFT`?

Reply:
I think "legacy" vs "non-legacy" or "legacy" vs "modern" is fine. These key formats are not really related to the consensus era (PBFT etc) they just happened to change at the same time, so I don't think PBFT-based naming will be helpful.

So these names are fine.